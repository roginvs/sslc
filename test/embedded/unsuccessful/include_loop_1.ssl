#include "include_loop_2.ssl"
