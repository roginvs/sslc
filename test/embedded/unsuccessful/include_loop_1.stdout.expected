Set include directory: ../include
Compiling include_loop_1.ssl
[Error] </unsuccessful/include_loop_2.ssl>:1: More than 256 nesting of #include
        from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from /unsuccessful/include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
    from /unsuccessful/include_loop_2.ssl: 1:    #include "include_loop_1.ssl"
    from include_loop_1.ssl: 1:    #include "include_loop_2.ssl"
1 error in preprocessor.
*** An error occured during preprocessing of include_loop_1.ssl ***
